Modules/System/Cues and KPIs/src/CuesAndKPIsImpl.Codeunit.al

```diff
@@ -121,7 +121,12 @@ codeunit 9702 "Cues And KPIs Impl."
     local procedure GetCustomizedCueStyleOption(TableId: Integer; FieldNo: Integer; CueValue: Decimal): Integer
     var
         CueSetup: Record "Cue Setup";
+        CueStyle: Enum "Cues And KPIs Style";
+        Resolved: Boolean;
     begin
+        CuesAndKPIs.OnBeforeGetCustomizedCueStyleOption(TableID, FieldNo, CueValue, CueStyle, Resolved);
+        if Resolved then
```

Comment:
The pattern is somewhere called `Handled`. Not that I mind the `Resolved` version, but what made you change your mind?

Reply:
I do not have anything against naming it "Handled". I saw other publisher in Codeunit 9701 called OnConvertStyleToStyleText and the term "Resolved" and it sounded better than Handled at least in the case for this purpose in this context.